tests/struct_method_parameters.yaml

```diff
@@ -37,8 +37,8 @@ templates:
         p1.y = 7;
 
         point p2;
-        p2.x = {{ p2.x }};
-        p2.y = {{ p2.y }};
+        p2.x = {{ p2x }};
```

Comment:
The dyaml implementation seemed to have a problem with keys with dots in them. It may have treated it as an accessor and expected p2 to be a dictionary. Maybe I was supposed to have parsed the p2 defined above as a dictionary too? Idk

Reply:
I think the appropriate way to set the key values would have been to do `p2: { x: blah, y: blah2 }` rather than `p2.x` in order to use `p2.x` in the output. But I don't know yaml that well.